mycroft/audio/interface.py

```diff
@@ -79,14 +88,24 @@ def queue(self, tracks=None):
             tracks = [tracks]
         elif not isinstance(tracks, list):
             raise ValueError
-        tracks = [ensure_uri(t) for t in tracks]
-        msg = self._format_msg('mycroft.audio.service.queue',
```

Comment:
Is there any reason to also emit this message for third party listeners? I'd assume not unless this is documented somewhere..

Reply:
no, "old style" audio service messages are only emitted by OCP as needed

they are also handled by ocp plugin in case they are emitted by a skill (happens in mycroft-core where this new code is skipped)

this PR aims to remove all mycroft.audio.service messages to the OCP package where they are handled

Reply:
Makes sense with the other PR's. I guess audio services in general are core/backend-specific to a degree and have to be for compat./feature support